lib/mix/lib/mix/compilers/elixir.ex

```diff
@@ -313,7 +332,8 @@ defmodule Mix.Compilers.Elixir do
   end
 
   # Similar to read manifest but supports data migration.
-  defp parse_manifest(manifest) do
+  @doc false
```

Comment:
The whole module is `@moduledoc false`, so we can move this up and add `@doc` to it.